src/server/config/schema.js

```diff
@@ -186,7 +186,7 @@ export default () => Joi.object({
         then: Joi.default(!process.stdout.isTTY),
         otherwise: Joi.default(true)
       }),
-    timezone: Joi.string().allow(false).default('UTC')
+    timezone: Joi.string().allow(false).default(false)
```

Comment:
there's nothing magic with false, basically saying don't set it.  maybe i should null it